### PR TITLE
[Docs] Consoleify multi-search and search-template docs

### PR DESCRIPTION
**Pull request by tlrx**
Relates #23001

### CODE REVIEW

File: docs/build.gradle [hunk 1]
```diff
@@ -132,11 +132,10 @@ buildRestTests.expectedUnconvertedCandidates = [
   'reference/query-dsl/function-score-query.asciidoc',
   'reference/query-dsl/geo-shape-query.asciidoc',
   'reference/search/field-stats.asciidoc',
-  'reference/search/multi-search.asciidoc',
   'reference/search/profile.asciidoc',
   'reference/search/request/highlighting.asciidoc',
   'reference/search/request/inner-hits.asciidoc',
-  'reference/search/search-template.asciidoc',
+  'reference/search/search-template.asciidoc', // We don't want to consoleify the rendered templates in this doc
```
Comment (nik9000) [inline docs/build.gradle]: Mark them as `// NOTCONSOLE` then. That means that someone has looked at them and is sure they shouldn't be converted.

File: docs/reference/search/multi-search.asciidoc [hunk 2]
```diff
@@ -90,26 +96,58 @@ also provides support for templates. Submit them like follows:
...
+    "template": {
+        "query": {
+            "match": {
+                "message": "{{query_string}}"
+            }
+        }
+    }
+}
+------------------------------------------
+// CONSOLE
```
Comment (nik9000) [inline docs/reference/search/multi-search.asciidoc]: If you setup twitter here and then `// TEST[continued]` on the next two snippets then you'll get a "more real" test, I think.

File: docs/reference/search/search-template.asciidoc [hunk 3]
```diff
@@ -110,18 +118,21 @@ which is rendered as:
...
           {
-            "term" : "foo"
+            "term" : {
+                "user" : "foo"
+            }
           },
           {
-            "term" : "bar"
+            "term" : {
+                "user" : "bar"
```
Comment (nik9000) [inline docs/reference/search/search-template.asciidoc]: ❤️ 

I love finding mistakes, even if they are small.

File: docs/reference/search/search-template.asciidoc [hunk 4]
```diff
@@ -415,49 +427,66 @@ POST /_search/template/<templatename>
     }
 }
 ------------------------------------------
+// CONSOLE
+// TEST[continued]
+
+//////////////////////////
+
```
Comment (nik9000) [inline docs/reference/search/search-template.asciidoc]: Probably worth adding a short comment. At this point we mostly understand why we have these but it is nice to have the comment anyway.

**Review by nik9000 — APPROVED** [review]
Left a few minors but good with me.

File: docs/build.gradle (hunk 1 above)
Comment (tlrx) [inline docs/build.gradle]: But rendered templates will have "view in console" buttons and "copy as curl" which does not make sense...?

**Pull request merged**